packages/main/src/Popover.js

```diff
@@ -336,17 +337,21 @@ class Popover extends Popup {
 		this.show();
 	}
 
-	show() {
+	async show() {
 		let placement;
-		const popoverSize = this.popoverSize;
-		const openerRect = this._opener.getBoundingClientRect();
+		const popoverSize = await this._popoverSize;
```

Comment:
I find this quite unintuitive, as this getter will create a new promise each time, where as I expect a promise was assigned to `this` and once resolved will always be in this state.

Maybe make this a getPopoverSize() method instead?